scripts/create_20MB_random_asset_file.py

```diff
@@ -0,0 +1,8 @@
+# this will create a 20 MB dummy asset file for testing the upload via
+# the admin GUI.
+
+import os
+
+FILE_SIZE = 20 * 1024**2  # this is 20 MB
+with open("20MB_sample_asset_file.zip", "wb") as dummy_file:
+    dummy_file.write(os.urandom(FILE_SIZE))
```

Comment:
You use `zip` extension but this is not a valid zip file which could be misleading while testing.  Might be a good idea to really zip the file to have a valid zip file.

Reply:
The file size could/should be an option parameter to the script and the size should not be part of the file name. The file should also be in the `.gitignore` because it doesn't needs to be under source control. Moreover NEVER put big binary files under source control unless it is absolutely necessary. This will in future especially when the file changes have an impact on git performance !

Reply:
Makes sense, thanks!